plugin-dev/Source/Sentry/Public/SentryEvent.h

```diff
@@ -19,6 +19,19 @@ class SENTRY_API USentryEvent : public UObject
 public:
 	USentryEvent();
 
+	/** Creates the empty event. */
+	UFUNCTION(BlueprintCallable, Category = "Sentry")
+	static USentryEvent* CreateEmptyEvent();
```

Comment:
```suggestion
	static USentryEvent* CreateEvent();
```
It's unfortunate that we can't overload functions for blueprints.

Reply:
After giving it a second thought I guess we might actually remove that completely and simply create a new event using `NewObject` (in CPP) and `ConstructObject` (in BP)